packages/rum-vue/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@elastic/apm-rum-vue",
-  "version": "1.3.1",
+  "version": "1.3.2",
```

Comment:
will publish it as 2.0.0 as its breaking change and also to keep 1.x focussed on prior vue versions.

Reply:
Switched to 1.3.1 as before, will be changed by our release script when doing major bump.